source/Government.cpp

```diff
@@ -258,7 +263,26 @@ void Government::Load(const DataNode &node)
 		else if(key == "language")
 			language = child.Token(valueIndex);
 		else if(key == "raid")
-			raidFleet = GameData::Fleets().Get(child.Token(valueIndex));
+		{
+			if(notClearedRaids && !add && !remove)
```

Comment:
There's code above this point that prevents `add` from ever being true by the time we reach here:
```
else if(add)
	child.PrintTrace("Error: Unsupported use of add:");
```